ReactCommon/react/renderer/templateprocessor/UITemplateProcessor.cpp

```diff
@@ -61,13 +61,13 @@ SharedShadowNode UITemplateProcessor::runCommand(
       LOG(INFO)
           << "(stop) UITemplateProcessor inject serialized 'server rendered' view tree";
     }
-    return nodes[command[1].asInt()];
+    return nodes[static_cast<Tag>(command[1].asInt())];
   } else if (opcode == "loadNativeBool") {
-    int registerNumber = command[1].asInt();
+    auto registerNumber = static_cast<size_t>(command[1].asInt());
     std::string param = command[4][0].asString();
     registers[registerNumber] = reactNativeConfig->getBool(param);
   } else if (opcode == "conditional") {
-    int registerNumber = command[1].asInt();
+    auto registerNumber = static_cast<size_t>(command[1].asInt());
```

Comment:
assert that these are >=0 as signed ints?

Reply:
Similar to above, the index will be checked by the vector, which will throw.